src/main/java/com/nitrobox/lombokbuilderhelper/LombokBuilderInspectionAll.java

```diff
@@ -33,21 +35,20 @@ public class LombokBuilderInspectionAll extends AbstractLombokBuilderInspection
 
 
     private List<String> getAllFields(PsiClass aClass) {
-        final String defaultBuilderValueAnnotation = "lombok.Builder.Default";
-        return Arrays.stream(aClass.getAllFields()).filter(field -> {
-            final PsiAnnotation[] annotations = field.getAnnotations();
-            final PsiModifierList modifiers = field.getModifierList();
-            final boolean isStaticField =
-                    modifiers != null && modifiers.hasModifierProperty(PsiModifier.STATIC);
-            return !isStaticField && Arrays.stream(annotations)
-                    .noneMatch(annotation -> Objects.equals(
-                            annotation.getQualifiedName(),
-                            defaultBuilderValueAnnotation));
-        }).map(PsiField::getName).filter(equalsString("id").negate()).filter(equalsString("createdTimestamp").negate()).toList();
+        return Arrays.stream(aClass.getAllFields()).filter(this::filterField).map(PsiField::getName).toList();
```

Comment:
src/main/java/com/nitrobox/lombokbuilderhelper/LombokBuilderInspection.java has to be changed too